ci-operator/step-registry/baremetalds/assisted/operator/publish/baremetalds-assisted-operator-publish-commands.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted operator publish command ************"
+
+UNRELEASED_SEMVER="99.0.0-unreleased"
+PROJECT_ROOT="$(readlink -e "$(dirname "${BASH_SOURCE[0]}")"/../)"
+OPERATOR_MANIFESTS="${PROJECT_ROOT}/deploy/olm-catalog/manifests"
+OPERATOR_METADATA="${PROJECT_ROOT}/deploy/olm-catalog/metadata"
+CSV="assisted-service-operator.clusterserviceversion.yaml"
+OPERATOR_VERSION=${OPERATOR_VERSION:-}
+
+GITHUB_USER="${GITHUB_USER:-}"
+GITHUB_TOKEN="${GITHUB_TOKEN:-}"
+GITHUB_NAME="${GITHUB_NAME:-}"
+GITHUB_EMAIL="${GITHUB_EMAIL:-}"
```

Comment:
It is looking like this will need to be configured in the CI so these env vars need to be added as a parameter in the `-ref.yaml`

For the rest it looks like they are ok because the script is populating them. But maybe `OPERATOR_VERSION` is also something we want to configure?